app/views/controllers/names/show/_classification.html.erb

```diff
@@ -3,40 +3,54 @@ approved_name = @name.approved_name
 parents = approved_name.all_parents
 %>
 
-<% if approved_name.classification.present? && parents.any? %>
-  <div class="mb-2">
-    <% ([approved_name] + parents).reverse.each do |n| %>
-      <p>
-        <%= rank_as_string(n.rank) %>:
-        <i><%= link_with_query(n.text_name.t, n.show_link_args) %></i>
-        <% if n == approved_name && approved_name != @name %>
-          <br/>&nbsp;&nbsp;(= <i><%= @name.text_name.t %></i>)
-        <% end %>
-      </p>
-    <% end %>
-  </div>
-<% end %>
+<%= if approved_name.classification.present? && parents.any?
+  tag.div(class: "mb-2") do
+    ([approved_name] + parents).reverse.each do |n|
+      concat(tag.p do
+        concat("#{rank_as_string(n.rank)}: ")
+        concat(tag.i(link_with_query(n.text_name.t, n.show_link_args)))
+        if n == approved_name && approved_name != @name
+          concat([
+            safe_br, "&nbsp;&nbsp; (= ", tag.i(@name.text_name.t), ")"
```

Comment:
```suggestion
            safe_br, safe_nbsp, safe_nbsp, " (= ", tag.i(@name.text_name.t), ")"
```
Fixes this issue https://github.com/MushroomObserver/mushroom-observer/pull/1774#issuecomment-1881733372

Reply:
Thanks!